py4DSTEM/braggvectors/diskdetection_aiml.py

```diff
@@ -918,7 +925,8 @@ def _get_latest_model(model_path=None):
         else:
             print("Checking the latest model on the cloud... \n")
             filename = file_path + file_type
-            download_file_from_google_drive(file_id, filename)
+            filename = Path(filename)
+            gdrive_download(file_id, destination="./tmp", filename=filename.name)
             try:
                 shutil.unpack_archive(filename, "./tmp", format="zip")
```

Comment:
I think it would be better to create a proper temporary directory with [`tempfile`](https://docs.python.org/3/library/tempfile.html) rather than making folders in the working directory.

Reply:
This definitely isn't the best way to handle it, but this pr was just to restore functionality.